lib/page/missing_country.rb

```diff
@@ -0,0 +1,8 @@
+module Page
+  class MissingCountry
+    attr_reader :missing
+    def initialize(country, world)
+      @missing = world[country.to_sym]
```

Comment:
Ah, good point. But rather than working around that, I suspect this is another perfect place for making the change easy so you can make the easy change. I suggest we harmonise the interface of both `country` at least to the extent of the layout template being able to call what _it_ needs on either. 

As a step towards that I've broken out a World class in https://github.com/everypolitician/viewer-sinatra/pull/14627 to encapsulate working with that file (as we're going to need that anyway). I suspect we should next wrap up each key/value pair in that as a simple Struct with the required interface.